<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySettingClass</class>
 <widget class="QWidget" name="DisplaySettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_display">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Rotation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Scale">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_displayInfo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>混合</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Src:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dst:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_src">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_dst">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_blendNormal">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>正常</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_blendAdditive">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>叠加</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_RotationVar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_ScaleVar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_atlas">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>230</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>图片列表</string>
   </property>
   <widget class="QListWidget" name="listWidget_atlas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_setFrameAnim">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设置为帧动画</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>550</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>帧列表</string>
   </property>
   <widget class="QPushButton" name="pushButton_deleteFrame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_frames">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveDown">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下移</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>帧率：</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_animateSpeed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_animateSpeed">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flashFrame">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_armature">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>900</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>动画列表</string>
   </property>
   <widget class="QListWidget" name="listWidget_armature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
